src/twisted/trial/util.py

```diff
@@ -240,8 +241,12 @@ def suppress(action="ignore", **kwarg):
 
 # This should be deleted, and replaced with twisted.application's code; see
 # #6016:
```

Comment:
very minor comment.

Maybe replace it with the full link https://github.com/twisted/twisted/issues/6016

not sure what is best.